from django.core.exceptions import ValidationError
import re

def validate_display_name(display_name):
    pass